CMakeLists.txt

```diff
@@ -29,12 +39,24 @@ target_include_directories(appx
                            PrivateHeaders
                            ${OPENSSL_INCLUDE_DIR}
                            ${ZLIB_INCLUDE_DIRS})
-target_link_libraries(appx
+						   
+						   
+if(STATIC_BUILD)
+ target_link_libraries(appx
+                      PRIVATE
+                      ${OPENSSL_LIBRARIES}
+                      ${ZLIB_LIBRARIES} -ldl -static-libgcc -static-libstdc++)
```

Comment:
Do you need the `-static-libgcc -static-libstdc++` explicitly? Doesn't passing `-static` achieve that automatically?